releases/release-1.11/exceptions.yaml

```diff
@@ -0,0 +1,17 @@
+# Exceptions to code freeze requested in 1.11
+# Google Group: https://groups.google.com/forum/#!topic/kubernetes-milestone-burndown
+# Release Team Lead: Josh Berkus (@jberkus)
+
+- name: "Update Istio addon manifest"
+  issue: "https://github.com/kubernetes/kubernetes/issues/64563"
```

Comment:
Needs to be tracked under the k/features repo.

Reply:
please explain, @idvoretskyi. This is intended as a point of record for exceptions to code freeze which have been requested with their resolution

Reply:
That seems independent of whether they had a feature issue open or not although they _should_ have